flow/Error.cpp

```diff
@@ -28,11 +28,6 @@ using std::make_pair;
 
 bool g_crashOnError = false;
 
-std::map<int, int>& Error::errorCounts() {
-	static std::map<int, int> counts;
-	return counts;
-}
-
```

Comment:
This is also referenced in some commented out code on line 73 that could be deleted. It is defined in Error.h:61, and a comment at Error.h:63 refers to it as well.